src/libstd/sys/unix/ext.rs

```diff
@@ -220,6 +224,27 @@ pub mod fs {
             self.as_inner_mut().mode(mode); self
         }
     }
+
+    /// Creates a new symbolic link on the filesystem.
+    ///
+    /// The `dst` path will be a symbolic link pointing to the `src` path.
```

Comment:
These docs are the ones that primarily show up (because our doc builders are linux), so could you expand this a bit to include the information about Windows as well? Just something along the lines of "If you're on windows...".